Messages.py

```diff
@@ -955,9 +925,7 @@ def shuffle_messages(messages, except_hints=True, always_allow_skip=True):
     def is_exempt(m):
         hint_ids = (
             GOSSIP_STONE_MESSAGES + TEMPLE_HINTS_MESSAGES + LIGHT_ARROW_HINT +
-            list(KEYSANITY_MESSAGES.keys()) + shuffle_messages.shop_item_messages +
-            shuffle_messages.scrubs_message_ids +
```

Comment:
This looks like a conflict; did you mean to remove scrubs, chickens, and poes?

Reply:
I don't even recall making that change (nor catching it in my own audit before I submitted the PR), so that was definitely unintended.